### PR TITLE
Recreate the composer autoloader files

### PR DESCRIPTION
**Pull request by rullzer**
- Adds missing files (so they are statically loaded)

CC: @nickvergessen @LukasReschke @MorrisJobke @icewind1991

### CODE REVIEW

**Comment by icewind1991** [comment]
:+1: 

(this seems like something that should be automated...)

File: lib/composer/composer/autoload_namespaces.php [hunk 1]
```diff
@@ -1,25 +1,5 @@
...
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
- *
- * You should have received a copy of the GNU Affero General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- *
- */
```
Comment (nickvergessen) [inline lib/composer/composer/autoload_namespaces.php]: Should change build/license.php to not touch those files:
https://github.com/nextcloud/server/blob/b64c6d6993ca30882f7c7e06344e1b7de34ca817/build/license.php#L372-L372
replace with `lib/public` and `lib/private`

**Comment by nickvergessen** [comment]
:+1: for automation

File: lib/composer/composer/autoload_namespaces.php (hunk 1 above)
Comment (rullzer) [inline lib/composer/composer/autoload_namespaces.php]: yes I'll do that

**Comment by rullzer** [comment]
I agree. I'll write a simple script that we can run daily as a start.

**Comment by LukasReschke** [comment]
> I agree. I'll write a simple script that we can run daily as a start.

Maybe just add it to the CI execution?

**Comment by MorrisJobke** [comment]
> Maybe just add it to the CI execution? So if the file is outdated => fail build?

makes sense :+1:

Better than a weird commit somewhere in between the git history that maybe also can cause a CI failure.

**Pull request merged**